bin/freshli-agent-mock.rb

```diff
@@ -15,7 +15,7 @@
 
 case ARGV[0]
 when 'validating-package-urls'
-  puts 'pkg:nuget/org.corgibytes.flyswatter/flyswatter'
+  puts 'pkg:nuget/org.corgibytes.flyswatter/flyswatter@1.1.0 1990-01-29T12:15:25-1:00'
```

Comment:
I think for the tests to keep working we probably have to add the other two as well:
```
pkg:nuget/org.corgibytes.calculatron/calculatron@21.3 2022-10-16T00:00:00+0:00
pkg:nuget/org.corgibytes.calculatron/calculatron@14.6 2019-12-31T00:00:00+0:00
```